supertokens_python/recipe/dashboard/interfaces.py

```diff
@@ -14,15 +14,15 @@
 from __future__ import annotations
 
 from abc import ABC, abstractmethod
-from typing import TYPE_CHECKING, Any, Dict, Callable, Awaitable, Optional, List
+from typing import TYPE_CHECKING, Any, Awaitable, Callable, Dict, List, Optional
 
 from ...supertokens import AppInfo
-
+from ...types import APIResponse, User
 from .utils import DashboardConfig
-from ...types import User, APIResponse
 
 if TYPE_CHECKING:
     from supertokens_python.framework import BaseRequest, BaseResponse
+    from supertokens_python.recipe.session.interfaces import SessionInformationResult
```

Comment:
Dont use types from recipes, redefine the types to exactly match the API specs

Reply:
Which one are you exactly talking about?

Reply:
SessionInformationResult and User

Reply:
I did for user using `UserWithMetadata` 

But for `SessionInformationResult` I don't see a new class adds value. Am I missing something?

Reply:
The idea is that the dashboard APIs always follow the spec and as and when the SDK logic/types change we are conscious about the impact it has on the dashboard APIs.

Currently if the `SessionInformationResult` type changes it will automatically reflect in the dashboard API which may or may not break the frontend logic.